OpenRA.Mods.Common/Traits/FireWarheads.cs

```diff
@@ -17,7 +17,7 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Detonate defined warheads at the current location at a set interval.")]
-	public class FireWarheadsInfo : PausableConditionalTraitInfo, Requires<IMoveInfo>, IRulesetLoaded
+	public class FireWarheadsInfo : PausableConditionalTraitInfo, IRulesetLoaded
```

Comment:
I guess that the `IMoveInfo` was here to make sure that `Actor.CenterPosition` will be defined. This should change to `IOccupySpaceInfo` instead.